### ISSUE TITLE
Changing only the CHANGELOG.md file doesn't trigger the MessageSkipBuildTag rule

### ISSUE THREAD

**Issue by tombruijn** [lintje/lintje#48]
Not sure what happens here:

```
$ lintje --version
lintje 0.8.0

$ lintje 
[DEBUG] No issues found for commit '8c166d5c75dcc74ed2d93d97d467b16ee53dac3f' in rule 'MessageSkipBuildTag'
[DEBUG] Commits: [Commit { long_sha: Some("8c166d5c75dcc74ed2d93d97d467b16ee53dac3f"), short_sha: Some("8c166d5"), email: Some("tom@tomdebruijn.com"), subject: "URL detection ticket references is a addition", message: "\nIn the CHANGELOG mark the URL detection and better detection for the\nrepo shorthands as a new feature rather than a change. It does more now.\n", file_changes: ["CHANGELOG.md"], issues: [Issue { type: Hint, rule: MessageTicketNumber, message: "The message body does not contain a ticket or issue number", position: MessageLine { line: 6, column: 1 }, context: [Context { type: Plain, line: Some(4), content: "repo shorthands as a new feature rather than a change. It does more now.", range: None, message: None }, Context { type: Plain, line: Some(5), content: "", range: None, message: None }, Context { type: Addition, line: Some(6), content: "Fixes #123", range: Some(0..10), message: Some("Consider adding a reference to a ticket or issue") }] }], ignored: false, ignored_rules: [] }]
```

**Comment by tombruijn** [comment]
I forgot I didn't configure this for CHANGELOG.md because some builds may check is the file is valid or not.
